sound/soc/sof/topology.c

```diff
@@ -804,6 +817,70 @@ static int sof_connect_dai_widget(struct snd_soc_component *scomp,
 	return 0;
 }
 
+/* LBM kcontrol, used to control loopback mode on SSP */
+static int sof_dai_add_lbm_kcontrol(struct snd_soc_component *scomp,
+				    u32 comp_id, u32 index)
+{
+	struct snd_sof_dev *sdev = snd_soc_component_get_drvdata(scomp);
+	struct snd_sof_control *scontrol;
+	struct sof_ipc_ctrl_data *cdata;
+	struct snd_kcontrol *kcontrol;
+	struct snd_kcontrol_new *kc;
+	char kcname[50];
```

Comment:
please align the size with ALSA  asounds.h/control.h

Reply:
Will check that. Thanks